### PR TITLE
Initial Delivery of Ingestion Tests to health-patterns

### PR DESCRIPTION
**Pull request by rguderian**
Signed-off-by: Roger Guderian <roger.guderian@ibm.com>

### CODE REVIEW

File: ingest/pom.xml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      <version>5.0.1</version>
+    </dependency>
+  </dependencies>
+    <build>
+    <finalName>${project.artifactId}</finalName>
+    <testOutputDirectory>target/test-classes</testOutputDirectory>
+    <testSourceDirectory>src/test/java</testSourceDirectory>
+    <plugins>   
+    <plugin>
+                    <groupId>com.google.code.maven-replacer-plugin</groupId>
```
Comment (ranum) [inline ingest/pom.xml:48]: nit-indentation seems excessive

File: ingest/src/test/java/tests/Ingestion_Initialize.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import org.junit.runner.RunWith;
+
+import categories.BasicIngestionInit;
+
+@TargetEnv("clinical-ingestion-flow.properties")
+@RunWith(ZeroCodeUnitRunner.class)
+
+public class Ingestion_Initialize {
+
+//	 @Test
```
Comment (ranum) [inline ingest/src/test/java/tests/Ingestion_Initialize.java:28]: Is this comment needed anymore?

**Review by ranum — APPROVED** [review]
LGTM

File: ingest/pom.xml (hunk 1 above)
Comment (rguderian) [reply ingest/pom.xml]: Fixed the indentations in the file.

File: ingest/src/test/java/tests/Ingestion_Initialize.java (hunk 2 above)
Comment (rguderian) [reply ingest/src/test/java/tests/Ingestion_Initialize.java]: Correct, the comment is no longer needed.   It has been removed.

**Pull request merged**